### ISSUE TITLE
frac_diff_ffd AttributeError

### ISSUE THREAD

**Issue by arthurire** [hudson-and-thames/mlfinlab#475]
**Describe the bug**

frac_diff_ffd declares a pd.Series as input but actually requires a pd.DataFrame

**To Reproduce**
```
from mlfinlab.features.fracdiff import frac_diff_ffd
import pandas as pd
import numpy as np
data=pd.Series(np.ones(100))
frac_diff_ffd(data,0.5)
```
then the program raise:

`AttributeError: 'Series' object has no attribute 'columns'`

**Expected behavior**
should be `name = series.name`

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - No
 - 1.5.0

**Comment by PanPip** [comment]
Hi @arthurire!

Thank you for reporting this issue. Indeed the expected input is of a `pd.DataFrame` type. This issue will be fixed in the new version release of the MlFinLab.

**Comment by PanPip** [comment]
This issue is now fixed in MlFinLab v 0.15.1.
